lib/xendit_api/api/virtual_account.rb

```diff
@@ -6,12 +6,12 @@ module Api
     class VirtualAccount < XenditApi::Api::Base
       PATH = '/callback_virtual_accounts'.freeze
 
-      def create(params)
+      def create(params, headers = nil)
         params[:is_closed] = true if params[:is_closed].nil?
         params[:is_single_use] = true if params[:is_single_use].nil?
         params[:expected_amount] = params[:amount] unless params[:amount].nil?
 
-        response = client.post(PATH, params)
+        response = client.post(PATH, params, headers)
```

Comment:
If you already specified headers = nil , does this headers also nil in here? Or it depends on the function that uses it? If the function in mekari payment use this and add headers with value, Does the headers in this also will using those value?

Reply:
I think we can set empty hash for default value of headers

Reply:
headers = {} ya you mean sal

Reply:
yes @okfriansyah-moh